kornia/geometry/warp/homography_warper.py

```diff
@@ -314,9 +318,14 @@ def normal_transform_pixel3d(depth: int, height: int, width: int) -> torch.Tenso
                            [0.0, 0.0, 1.0, -1.0],
                            [0.0, 0.0, 0.0, 1.0]])  # 4x4
 
-    tr_mat[0, 0] = tr_mat[0, 0] * 2.0 / (width - 1.0)
-    tr_mat[1, 1] = tr_mat[1, 1] * 2.0 / (height - 1.0)
-    tr_mat[2, 2] = tr_mat[2, 2] * 2.0 / (depth - 1.0)
+    # prevent divide by zero bugs
+    width_denom: float = 1e-14 if width == 1 else width - 1.0
```

Comment:
I think we probably shall make eps as a parameter.

Reply:
Just pushed that change